client/src/i18n/locales/es/index.json

```diff
@@ -76,6 +76,19 @@
 
   "genericErrorMessage": "Algo salió mal, inténtalo de nuevo más tarde.",
 
+  "alreadyConfirmed": "Este correo electrónico ya ha sido confirmado, por favor intente iniciar sesión. Si ha olvidado tu contraseña, por favor restáurela abajo",
+  "confirmationPeriodExpired": "Tu período de confirmación ha expirado",
+  "genericEmailConfirmationError": "Hemos tenido un problema para confirmar tu correo electrónico",
+  "yourConfirmationToken": "Tu símbolo de confirmación es",
+  "blank": "en blanco",
+  "invalid": "inválido",
+  "mustUseConfirmationEmail": "Debe confirmar tu cuenta utilizando un enlace de confirmación enviado a tu dirección de correo electrónico",
```

Comment:
"mustUseConfirmationEmail": "Tienes que confirmar tu cuenta utilizando un enlace de confirmación enviado a tu correo",

Reply:
@katelovescode flagging that this needs to have a period at the end of it. Seeing the earlier version of this translation come up w/o one in my QA of #93